menhir-recover/recovery_intf.ml

```diff
@@ -12,23 +12,8 @@ module type RECOVERY = sig
   type item = G.lr1 * G.production * int
 
   type recovery = {
-    prefix: int;
     cases: (G.lr1 option * item list) list;
   }
-  (** [prefix] is the size of the known prefix of the stack.
-      It means that in the kernel of current state, there is an item whose dot
-      is at position [prefix].
-      (we know the incoming symbols for these stack frames and we can enumerate
-      the possible state numbers).
-
-      [cases] is a mapping that associates to each possible state found at
-      stack.[-prefix]
-      (or None if the stack is empty) a list of reductions to execute.
-
-      The actual list of actions to reduce an item [(state, prod, pos)] is
-      given by
-          [Synthesizer.solution (Trail (state, prod, pos))]
-    *)
 
```

Comment:
The first paragraph is outdated, because I removed `prefix` completely. The second paragraph is a little outdated, because it uses `prefix` a little. Third paragraph is not outdated

Reply:
Maybe we can make it actual? because I guess these pieces won't be deleted.

Reply:
Done